src/app/_hooks/useCategory.ts

```diff
@@ -12,14 +17,15 @@ export type UseCategoryProps<Entry extends Object> = {
   searchParams: NextServerSearchParams
   entries: Array<Entry>
   validCategoryIds: Array<CategoryId>
+  categoryOptions: Array<CategoryOption>
```

Comment:
Why do we need to pass an array of options with count if we calculate `count` in the hook?

```ts
export type CategoryOption = {
  id: CategoryId
  name: string
  count?: number
}
```

Reply:
Ah yes, we should change the type. The `categoryOptions` we pass are without count, they look something like this
```
 categoryOptions: [
    { id: 'ecosystem', name: 'Ecosystem' },
    { id: 'events', name: 'Events' },
    { id: 'interviews', name: 'Interviews' },
    { id: 'news', name: 'News' },
    { id: 'reports', name: 'Reports' },
    { id: 'use-cases', name: 'Use Cases' }
  ]
```

Reply:
✅